notebooks/official/tabular_workflows/prophet_on_vertex_pipelines.ipynb

```diff
@@ -50,7 +50,7 @@
         "    </a>\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    import os
</pre><p>Here's the lint error. Should be:</p><p> </p><pre class="ql-syntax" spellcheck="false">import json
import os
import urllib
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1944/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>